desktop/src/main/java/bisq/desktop/main/portfolio/closedtrades/ClosedTradesViewModel.java

```diff
@@ -20,11 +20,12 @@
 import bisq.desktop.common.model.ActivatableWithDataModel;
 import bisq.desktop.common.model.ViewModel;
 import bisq.desktop.util.DisplayUtils;
-
+import bisq.common.crypto.KeyRing;
```

Comment:
Empty line is missing before and after this import to be in line with our imports format